plugins/mcreator-core/blockly/js/extensions_procedure_deps.js

```diff
@@ -0,0 +1,225 @@
+Blockly.Extensions.register('procedure_dependencies_tooltip',
+    function () {
+        let thisBlock = this;
+        this.setTooltip(function () {
+            const depList = javabridge.getDependencies(thisBlock.getFieldValue('procedure'));
+            if (depList.length === 0)
+                return javabridge.t('blockly.extension.procedure_dep_tooltip.empty');
+            let tooltip = javabridge.t('blockly.extension.procedure_dep_tooltip');
+            for (const dependency of depList)
+                tooltip += '\n' + dependency;
+            return tooltip;
+        });
+    });
+
+Blockly.Extensions.register('procedure_dependencies_onchange_mixin',
+    function () {
+        this.setOnChange(function (changeEvent) {
+            // Trigger the change only if a block is changed, moved, deleted or created
+            if ((changeEvent.type !== Blockly.Events.BLOCK_CHANGE ||

```

Comment:
This looks different from the usual code, one example from other MCreator place:

![image](https://github.com/MCreator/MCreator/assets/16374228/3ebe5e05-accc-4f6a-bf49-dc537c27bb44)

My question is why? My additional question is if there is some sort of bug because the comment mentions deletion action, but deletion is omitted compared to the code linked above?

Reply:
It does not alight with either comment or code. Either needs to be changed.

Checking my screenshot, others have explicit check for delete and comment also mentions delete.

So currently the code is inconsistent either in the code or in the comments.

> letion seemed to get detected properly by this code last time I checked.

This sounds a bit weak argument to why it is not here but in all other places in the codebase

Reply:
Is this resolved?